wiki/Help_centre/Account/en.md

```diff
@@ -208,12 +208,26 @@ This doesn't mean you can make a new account to do the same thing, either! Havin
 
 ### Can I delete my account?
 
-*For more information on account deletion, see: [your rights and controls](/wiki/Legal/Privacy#your-rights-and-control).*
+*For more information on account deletion, see: [osu! privacy policy § Your rights and control](/wiki/Legal/Privacy#your-rights-and-control).*
 
 *Caution: Account deletion is permanent, and therefore cannot be undone!*
 
 Yes, any user can request for their account to be deleted by sending an email to [privacy@ppy.sh](mailto:privacy@ppy.sh). Keep in mind that deleting your account is **final**, meaning that it cannot be reversed or undone. Deleting your account does not mean you can create a new account, as having multiple accounts is against the [rules](/wiki/Rules#community-rules). This means you will no longer be able to use any of osu!'s online features.
 
+### I no longer want to keep playing, but I don't want my account to be deleted! 
+
+*For more information on what you can and cannot do with your account see: [osu! privacy policy § Your rights and control](/wiki/Legal/Privacy#your-rights-and-control).*
+
+Any user can request their account to be **locked** by sending an email to [privacy@ppy.sh](mailto:privacy@ppy.sh), [accounts@ppy.sh](mailto:accounts@ppy.sh), or [support@ppy.sh](mailto:support@ppy.sh). The account's associated password will then be scrambled, locking the user out of their account.
+
+All information from a locked account will still be publicly visible.
+
+To unlock the account again, simply send an email to one of the aforementioned addresses expressing the intent to unlock it.
+
+*This feature is not to be abused.* Locking an account should be one of the last resorts for those players that don't want to leave osu! forever, but **need** dedicated time away from osu! for a legitimate reason.
+
+If you do not have a valid reason to lock your account, you can change your password to random words or letters you will never remember and log out. This is essentially the same process as locking. Once you are ready to come back into the community, send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) requesting a password reset.
```

Comment:
```suggestion
You may lock yourself out of your account by changing your email and password to random words or letters you will never remember and logging out. This is essentially the same process as locking. Once you are ready to come back into the community, send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) requesting a password reset. Make sure you use the email address previously used by your account.
```
Support will not, or exceedingly rarely, ask users to justify their requests. There's no need to expand on one's reasons to get their accounts locked; they only need to make the request.